### PR TITLE
Add join group by groupInviteMessage

### PR DESCRIPTION
**Pull request by Schwartzblat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -732,6 +732,13 @@ Of course, replace ``` xyz ``` with an actual ID.
     ```
     Of course, replace ``` xxx ``` with invitation code.
 
+
+- To join the group using groupInviteMessage
+    ``` ts
+    const response = await sock.groupAcceptInviteV4(groupInviteMessage")
```
Comment (azudindaem) [inline README.md:738]: typo 
`const response = await sock.groupAcceptInviteV4("groupInviteMessage")`

**Review by azudindaem — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Schwartzblat) [reply README.md]: Actually it supposed to be without "" at all :(
Fixed.

**Comment by azudindaem** [comment]
there is no need to add new functionality to an existing one

**Comment by ferhacks** [comment]
.

**Review by thisfardi — APPROVED** [review]

**Pull request merged**